File: elixir/0009.exs
defmodule Solution do
  @spec is_palindrome(x :: integer) :: boolean
  def is_palindrome(x) do
    s = Integer.to_string(x)
    rev = String.reverse(s)
    s == rev
  end
end

x = Solution.is_palindrome(10)
IO.inspect x

File: dart/0009.dart
class Solution {
  bool isPalindrome(int x) {
    String s = x.toString();
    String t = s.split('').reversed.join();
    return t == s;
  }
}

void main() {
  Solution solution = Solution();
  print(solution.isPalindrome(12));
  print(solution.isPalindrome(11));
  print(solution.isPalindrome(1));
  print(solution.isPalindrome(121));
}